products/jbrowse-desktop/src/JBrowse.test.tsx

```diff
@@ -27,11 +28,15 @@ function getPluginManager(initialState?: SnapshotIn<JBrowseRootModel>) {
   const pluginManager = new PluginManager(corePlugins.map(P => new P()))
   pluginManager.createPluggableElements()
 
-  const JBrowseRootModel = JBrowseRootModelFactory(pluginManager)
+  const JBrowseRootModel = JBrowseRootModelFactory(
+    pluginManager,
+    sessionModelFactory,
+  )
```

Comment:
The change to how the root model is instantiated was done to break a dependency cycle between the root, session, and jbrowse model types